abci/example/kvstore/kvstore.go

```diff
@@ -125,23 +138,19 @@ func (app *Application) Info(context.Context, *types.InfoRequest) (*types.InfoRe
 		}
 	}
 
-	if app.useLanes {
-		return &types.InfoResponse{
-			Data:                fmt.Sprintf("{\"size\":%v}", app.state.Size),
-			Version:             version.ABCIVersion,
-			AppVersion:          AppVersion,
-			LastBlockHeight:     app.state.Height,
-			LastBlockAppHash:    app.state.Hash(),
-			LanePriorities:      app.lanePriorities,
-			DefaultLanePriority: app.lanes[defaultLane],
-		}, nil
+	var defaultLanePriority uint32
+	if len(app.lanes) > 0 {
```

Comment:
What happens if `app.lanes` is `nil`? Should this be 

```
if app.lanes != nil {

   defaultLanePriority = app.lanes[defaultLane] 
} 
```

Reply:
It's the same, but I'm changing it to make it more clear.